README.md

```diff
@@ -1 +1,64 @@
-# Bucketeer CLI
+# Bucketeer CLI &nbsp;[![Build Status](https://travis-ci.com/UCLALibrary/bucketeer-cli.svg?branch=main)](https://travis-ci.com/UCLALibrary/bucketeer-cli)
+
+Uploads CSV files to the [Bucketeer](https://github.com/UCLALibrary/bucketeer) service for processing.
```

Comment:
Should we mention that access to UCLA's bucketeer is limited to campus IPs, so off-campus folks (all of us these days) needs to use the VPN? Or maybe that should go in the `Usage` section below?

Reply:
Yes, good call